packages/edit-site/src/components/sidebar/global-styles-v2/screens/index.js

```diff
@@ -0,0 +1,6 @@
+export { ColorsElementScreen } from './colors-element';
```

Comment:
Nit: from a dev experience standpoint, is it valid to call these "screens"?

Reply:
I think Q was taking a lot of hints from React Native development generally (View and Text component for example) and the navigation matches a lot of how RN does things, including the concept of Screens.

Reply:
The thing that can create confusion here is that a "screen" usually takes a whole screen, while here it's just a section of the entire UI.

Reply:
Yeah, I'm not sure what to call it. A view?

Reply:
I like that better, although it's used for some other things (a `View` component) 😉  I guess it should be fine regardless since it's an isolated term within the global styles. The problem with isolated terms is that as a project matures, they end up across the entire codebase, and isolated terms end up being not so isolated anymore 😉

Reply:
💯 

How about `GlobalStylesNavigationScreenAndOrViewThing` 😂 Surely that won't leak into the rest of the project lol!

Reply:
😉 

On a serious note, I agree - `GlobalStylesScreen` or `GlobalStylesView` would suffice for me.

Reply:
I personally prefer `View` over `Screen` too.

I also think that we should come up with a solution that works for any sidebars (and "self-contained" pieces of UI), not just the Global Styles sidebar specifically.

I believe Q put some of these concepts behind the `Navigation` namespace, for example `NavigationScreen` etc